### PR TITLE
Second Try

### LINKED ISSUE #1057: A change in netcdf-c broke a test in netcdf-fortran
Current `master` is seeing a failure in the fortran tests.  This is the second time this has happened recently, so I'm going to (for now) turn fortran testing on as part of the travis CI.  We also need to fix the `ftst_rengrps` test which is now failing.   The fix may be in the C library or the Fortran library, depending on what caused the issue. I'm spooling up a fortran and C++ release (both of which are overdue).

### PR DESCRIPTION
**Pull request by WardF**
* Fixes #1057 

Got things turned around on my local system and rather than try to untangle it, lets just try this again.

### CODE REVIEW

File: include/netcdf.h [hunk 1]
```diff
@@ -512,10 +512,10 @@ EXTERNL const char *
 nc_strerror(int ncerr);
 
 /* Set up user-defined format. */
-typedef struct NC_Dispatch NC_Dispatch;   
+typedef struct NC_Dispatch NC_Dispatch;
```
Comment (edhartnett) [inline include/netcdf.h]: Happened again. ;-)

**Comment by DennisHeimbigner** [comment]
Looking at commit
I see that  NC_HAVE_RENAME_GRP was removed
and the failing test appears to involve group renaming.
Is this used in the fortran code anywhere?

**Comment by DennisHeimbigner** [comment]
I was right. NC_HAVE_RENAME_GRP is used in nf_lib.c.

**Comment by DennisHeimbigner** [comment]
So also is NC_HAVE_INQ_FORMAT_EXTENDED which is also removed in that
commit.

**Pull request merged**